### DISCUSSION TITLE
[GitHub小知识] 在PR的普通comment中用fix, close, resolve不会触发issues

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#41] [2️⃣ 聊项目]
<img width="1284" alt="image" src="https://user-images.githubusercontent.com/32434520/169643099-0a84acd8-a32b-4a28-83bc-d2ae71488415.png">

给大家分享一个新发现：

GitHub PR页面中，在普通comment里用fix, close, resolve不会触发关联issue；只有放到第一个comment，也就是提PR时填的Description里才能触发。

随后参考了[官方文档](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)，发现的确是这样的规则：

<img width="754" alt="image" src="https://user-images.githubusercontent.com/32434520/169643217-609af516-608e-4d9f-bd2e-2e1b7fb538d0.png">

官方文档中还有介绍如何一次性关联多个issues以及如何在commit message中关联issues，有兴趣的小伙伴可以去看看~

**Comment by will-ww** [comment]
很有价值，GitHub 有很多这样的细节与体验，我们可以多提出来交流~

**Comment by xgdyp** [comment]
这个就是说只有在create操作的时候才能触发关联issue是嘛，貌似我们数据库里应该也是create和comment有区别的

**Reply by tyn1998** [reply]
> 这个就是说只有在create操作的时候才能触发关联issue是嘛，貌似我们数据库里应该也是create和comment有区别的

后续edit那个PR Description也是可以触发的~

**Reply by zcc0077** [reply]
> > 这个就是说只有在create操作的时候才能触发关联issue是嘛，貌似我们数据库里应该也是create和comment有区别的
> 
> 
> 
> 后续edit那个PR Description也是可以触发的~

为啥我每次edit完了还是提示找不到pr的报错No linked issues found. Please add the corresponding issues in the pull request description.
